### DISCUSSION TITLE
C++ API header file install

### DISCUSSION THREAD

**Discussion by DaniloNoguera** [OpenFAST/openfast#782] [Q&A] [answered]
Hello, in `openfast/glue-codes/openfast-cpp/CMakeLists.txt` there is a small error, the path `src/SC.H` must be changed to `src/SC.h`, as in previous versions to avoid error. Thanks for attention.

**Comment by rafmudaf** [comment, marked as answer]
Hi @DaniloNoguera this has been fixed in the `dev` branch via https://github.com/OpenFAST/openfast/pull/768.
